packages/ui-core/theme/src/lib/components/theme-sidebar/theme-settings/components/theme-selector/theme-selector-image/theme-selector-image.component.scss

```
@@ -1,93 +1,169 @@
+  &:focus-visible {
+    outline: 2px solid nb-theme(color-primary-default);
+    outline-offset: 2px;
+    border-radius: $panel-row-radius;
+  }
 }
-.themes{
-  max-height: calc(100vh - 25rem);
-  overflow-y: auto;
+
+.theme-trigger-label {
+  // The value box beside it is a fixed 8.75rem, so the label is the only part
```

Stylelint (17.14.0)

[error] Unknown word The (CssSyntaxError)

(CssSyntaxError)